SeaBattle/HybridShip.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class HybridShip : BaseShip, IAttack, IRepair
+    {
+        public HybridShip(string Name, List<Point> points)
+        {
+            var Decks = new List<Deck>();
+
+            for (int i = 0; i < points.Count - 1; i++)

```

Comment:
why is logic is duplicated in every ship?

Reply:
yeah, please move it to the base class. Thats what I expected for

Reply:
Fixed now.